src/components/Image.js

```diff
@@ -76,6 +76,12 @@ export const Image = ({
         break effect;
       }
 
+      if (__DEV__ && device.platform === 'android') {
+        setSource({ uri: sourceProp.uri });
+
+        break effect;
```

Comment:
please add a comment like in the prior condition

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/3fb7816d8ba405edd24837aee5e3466695dd3679